where-to-travel/src/main/webapp/script.js

```diff
@@ -246,8 +280,10 @@ function submitDataListener(event) {
   if (home == null) {
     const content = '<p> No home location found. Please set a home location and try again.</p>';
     openModal(content);
-  }
-  else {
+  } else if ($('#' + FEEDBACK_ID).children().length >= 1) {
+    const content = '<p> Please enter valid search parameters</p>';
```

Comment:
nit: Extra space after <p>, is this intended?

Reply:
fixed with 44d9929